model/src/main/java/com/javahelp/model/userEntityTest/User.java

```diff
@@ -0,0 +1,54 @@
+package com.javahelp.model.userEntityTest;
+import com.javahelp.model.user.UserInfo;
+import com.javahelp.model.user.ClientUserInfo;
+import com.javahelp.model.user.UserType;
+import com.javahelp.model.user.ProviderUserInfo;
+
+public class User {
+    /**
+     * A class for a user entity
+     */
```

Comment:
The convention in Java is to have Javadocs above the method/class/field they pertain to rather than below. For example, this section should look like

```java
/**
 * A class for a user entity
 */
public class User {
```

Reply:
Thanks!